src/Components/Feature/FIlterBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+
+interface FilterBarProps {
+    handleSearch: (value: string) => void;
+    handleSort: (value: string) => void;
+    handleStatus: (value: string) => void;
+}
+
+const FilterBar = ({
+    handleSearch,
+    handleSort,
+    handleStatus,
+}:FilterBarProps) => {
+    return (
+        <div className="filter-bar">
+            <input
+                type="text"
+                placeholder="Search..."
+                onChange={(e) => handleSearch(e.target.value)}
```

Comment:
remove this inline handler for all use cases like : (e) => handleSearch(e.target.value)
instead of this you can pass the event directly (onChnage={handleSearch} or you can write helper function